-- Work around the current devel/py-setuptools in the ports tree, which does
-- not support the PEP 621 "license" / "license-files" keys or the
-- setuptools>=77.0 build-system requirement.
--- pyproject.toml.orig	2026-09-07 00:00:00 UTC
+++ pyproject.toml	2026-09-07 00:00:00 UTC
@@ -1,6 +1,6 @@
 [build-system]
 requires = [
-    "setuptools>=77.0",
+    "setuptools",
     "pybind11>=3.0",
 ]
 build-backend = "setuptools.build_meta"
@@ -25,8 +24,6 @@
 maintainers = [
   {name = "The icet developer team"}
 ]
-license = "MPL-2.0"
-license-files = ["LICENSE", "THIRD_PARTY_LICENSES.md"]
 readme = "README.rst"
 requires-python = ">=3.10"
 keywords = ["chemistry", "physics", "scientific"]
